pkg/rpc/client/client.go

```diff
@@ -40,6 +42,7 @@ type Client struct {
 	wif        *keys.WIF
 	balancerMu *sync.Mutex
 	balancer   request.BalanceGetter
+	cache      *cache
```

Comment:
Why a pointer? Is it initialized anywhere?

Reply:
Removed pointer, it's not necessary here.